Tangem/Modules/CardSettings/CardSettingsCoordinator.swift

```diff
@@ -52,9 +54,15 @@ extension CardSettingsCoordinator {
 
 extension CardSettingsCoordinator: CardSettingsRoutable {
     func openOnboarding(with input: OnboardingInput) {
+        let isSavingCards = !userWalletListService.isEmpty
```

Comment:
а это лушче передать из viewModel, просто флагом в метод
```suggestion
    func openOnboarding(with input: OnboardingInput, isSavingCards: Bool) {
}
```

Reply:
Сделал